client/blocks/reader-full-post/index.jsx

```diff
@@ -151,6 +157,8 @@ export class FullPostView extends React.Component {
 		KeyboardShortcuts.off( 'like-selection', this.handleLike );
 		KeyboardShortcuts.off( 'move-selection-down', this.goToNextPost );
 		KeyboardShortcuts.off( 'move-selection-up', this.goToPreviousPost );
+		// Remove WPiFrameResize listener.
+		this.stopResize();
```

Comment:
This should be `this.stopResize?()`. Otherwise it will crash when `this.stopResize` is not assigned a function, like when `this.postContentWrapper.current` is null.

Reply:
Good catch and TIL 👍 90c8709